dbms/src/Columns/ColumnAggregateFunction.cpp

```diff
@@ -244,6 +248,25 @@ size_t ColumnAggregateFunction::allocatedBytes() const
     return res;
 }
 
+size_t ColumnAggregateFunction::estimateByteSizeForSpill() const
```

Comment:
what the diff of `byteSize()`?

Reply:
`byteSize()` will always add all the memory in `pools`

Reply:
Better to add some comments for it here.

Reply:
Done